### PR TITLE
LPS-190119 - UI configuration to create the OSGi configuration

### PR DESCRIPTION
**Pull request by alvarosaugarlr**
[LPS-190119](https://liferay.atlassian.net/browse/LPS-190119)

Feature:
- A portlet has been created to include the behavioral logic in the Instance Settings configuration.
- A scheduler has been created to check the expiration of access tokens and notify administrators. 

### Tests
Manual tests are described on [this comment](https://liferay.atlassian.net/browse/LPS-190119?focusedCommentId=2501754) of the ticket

### CODE REVIEW

File: modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+		String defaultEmailFromName = "SCIM-Notification";
+
+		InternetAddress from = new InternetAddress(
+			defaultEmailFromName, defaultEmailFromAddress);
+
+		List<InternetAddress> bcc = TransformUtil.transform(
+			users,
+			user -> {
+				if (!StringUtil.equals(
+						user.getEmailAddress(), "test@liferay.com")) {
```
Comment (stian-sigvartsen) [inline modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java:148]: @alvarosaugarlr Why did you need to put this check for test@liferay.com ?

Comment (stian-sigvartsen) [reply modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java]: Perfect. Yes best not to duplicate that logic

**Comment by alvarosaugarlr** [comment]
This pr has been splitted to make easier the merge.  The first pr is - https://github.com/brianchandotcom/liferay-portal/pull/143431

**Comment by stian-sigvartsen** [comment]
This PR has been superceded by others and the UI is complete, so I am closing it.
